### PR TITLE
odML Editor for odML v1.4

### PR DESCRIPTION
**Pull request by digaru19**
A skeletal framework for odML editor, supporting odML v1.4

Major change includes the _Pseudo-Value wrapper_ for odML values, and the modifications that follow it.

### CODE REVIEW

File: odmlui/EditorTab.py [hunk 1]
```diff
@@ -48,6 +48,13 @@ def new(self, doc=None):
         self.document = doc
         self.file_uri = None
 
+    def parse_properties(self, odml_sections):
+        for i in odml_sections:
+            create_pseudo_values(i.properties)
+
+        for j in odml_sections:
+            self.parse_properties(j.sections)
+
```
Comment (jgrewe) [inline odmlui/EditorTab.py]: can't this be done in the same loop?

File: odmlui/Editor.py [hunk 2]
```diff
@@ -13,7 +13,7 @@
 import odmlui.treemodel.mixin
 from . import commands
 
-from odmlui.treemodel import PropertyModel, SectionModel
+from odmlui.treemodel import PropertyModel, SectionModel, value
```
Comment (jgrewe) [inline odmlui/Editor.py]: would ValueModel be an appropriate name?

**Review by achilleas-k — APPROVED** [review]
All looks good here. This has been open for a while. Are we good to merge @jgrewe ?

**Review by jgrewe — APPROVED** [review]
oh, yeah, it's good to go

**Pull request merged**